generate.go

```diff
@@ -42,8 +42,12 @@ func GetPublicIdentity(b64Identity string) (string, error) {
 		PublicSignatureKey  []byte `json:"public_signature_key,omitempty"`
 		PublicEncryptionKey []byte `json:"public_encryption_key,omitempty"`
 	}
-	publicIdentity := &anyPublicIdentity{}
-	err := Base64JsonDecode(b64Identity, publicIdentity)
+
+	var (
+		publicIdentity anyPublicIdentity
+	)
```

Comment:
Why is this should be better ?

Reply:
The only need with this variable is to be able to send a reference to it to `Base64JsonDecode`, there's no interest in having a (hard-to-read) pointer to a composite literal here. Having a zero value and sending a reference to the function is semantically better and easier to read.
I can make it a one-line `var publicIdentity anyPublicIdentity` if you like, I parenthesized it here mostly out of habit.